backend/src/schema/resolvers/posts.js

```diff
@@ -211,10 +225,68 @@ export default {
       })
       return emoted
     },
+    pinPost: async (_parent, params, context, _resolveInfo) => {
+      let pinnedPost
+      const { driver, user } = context
+      const session = driver.session()
+      const { id: userId } = user
+      let writeTxResultPromise = session.writeTransaction(async transaction => {
+        const deletePreviousRelationsResponse = await transaction.run(
+          `
+          MATCH ()-[previousRelations:PINNED]->(post:Post)
```

Comment:
Why not `User`?
```suggestion
          MATCH (:User)-[previousRelations:PINNED]->(post:Post)
```

Reply:
Cause it's not important what or who pinned it

Reply:
???
Why then in file `backend/src/schema/types/type/Post.gql`, line 19:

```
pinnedAt: String @cypher(
      statement: "MATCH (this)<-[pinned:PINNED]-(:User) WHERE NOT this.deleted = true AND NOT this.disabled = true RETURN pinned.createdAt"
    )
```

In my eyes it is a kind of code quality if we are as good as specific we can be. That’s why we are writing test to nail the things more down, as an example.

For sure at the moment only users can pin posts. But that only users can do something is true  for a lot of things. Only users can emote to posts, but we query for the label `User` as well like in file "backend/src/schema/resolvers/posts.js“, line 315:
```
        emotionsCount: '<-[related:EMOTED]-(:User)',
```

Reply:
Sure, I updated it already... 
I don't think it's important, but if you do, I'm happy to update it, as I showed with my actions.